### ISSUE TITLE
ViewZone screen cheese with inline problems

### ISSUE THREAD

**Issue by oliversalzburg** [Microsoft/vscode#36762]
- VSCode Version: Code 1.17.2 (b813d12980308015bcd2b3a2f6efa5c810c33ba5, 2017-10-16T13:59:46.104Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
vscode-eslint|dba|1.3.2
pgsql|dou|0.0.8
EditorConfig|Edi|0.11.1
JSDocTagComplete|Hoo|0.0.2
vscode-esformatter|how|1.5.2
vscode-puglint|mrm|2.3.0

(2 theme extensions excluded)

---

Steps to Reproduce:

1. Have a project with ESLint integration, or possibly anything else that creates "problems".
2. Open a file with a problem and press F8.
3. An annoying bar appears at the location of the "problem".
![image](https://user-images.githubusercontent.com/1658949/31900637-5fa052ce-b81f-11e7-84e7-465aac8ffd5b.png)
  I am completely clueless how this bar can be removed again. **Please**, someone tell me.
4. Fix the "problem". The bar stays.
![image](https://user-images.githubusercontent.com/1658949/31900676-880716a8-b81f-11e7-84cb-502a0f127a8f.png)

Bonus content:
1. Select all text in the file and delete it.
2. Press Ctrl+Z.

![image](https://user-images.githubusercontent.com/1658949/31900692-a08c6e8a-b81f-11e7-8ef5-5409a949bbac.png)

**Comment by Spown** [comment]
this bar is apparently called ["inline zone"](https://code.visualstudio.com/docs/editor/editingevolved#_errors-warnings). there is no setting to make it not to appear on F8. However if you navigate to the problem via double click on it in the Problems panel - the focus will change to the editor, the cursor'll jump to the location and no inline zone will be shown. To remove the bar just hit ESC.

So basically you are requesting a user setting to not to show it on F8, ie. traverse problems without popping "inline zone" up.

**Comment by oliversalzburg** [comment]
> To remove the zone just hit ESC.
Thank you! What a relieve! :D

Not sure what I'm requesting. I think it would be great if it was more apparent how to hide the inline zone again. Maybe with a UI element in the problems pane.

The rendering issue in the "Bonus Content" doesn't look right for sure though.

**Comment by dbaeumer** [comment]
@jrieken assigning to you because of the rendering bug in bonus content.

**Comment by jrieken** [comment]
I cannot reproduce the screen cheese. Please try again with latest stable and current insiders. Thanks

**Comment by oliversalzburg** [comment]
@jrieken Confirmed. No longer reproduces on stable.
